runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/DataflowPipelineTranslator.java

```diff
@@ -948,6 +950,18 @@ private <K, V> void groupByKeyHelper(
                 PropertyNames.IS_MERGING_WINDOW_FN,
                 !windowingStrategy.getWindowFn().isNonMerging());
           }
+
+          private boolean containsCountTrigger(Trigger trigger) {
```

Comment:
I realize that triggers are mostly considered a closed set these days, but if we ever go back to implement things like data-driven triggers those would need to be excluded as well, so I'd be more comfortable with an allow list of what can be lifted than a denylist of what can't. (A generic/default implantation of recursing into sub-triggers is probably fine.)